packages/dds/sequence/src/sharedNumberSequence.ts

```diff
@@ -7,6 +7,12 @@ import { IFluidDataStoreRuntime, IChannelAttributes } from "@fluidframework/data
 import { SharedNumberSequenceFactory } from "./sequenceFactory";
 import { SharedSequence } from "./sharedSequence";
 
+/**
+ * The SharedNumberSequence holds a sequence of numbers. Each number will be stored
```

Comment:
It's probably fine, but glancing at two links below, they do not use similar wording.
I think better wording would be talking about accessing elements and using indexes for that, not storing at position.

https://developer.mozilla.org/en-US/docs/web/javascript/reference/global_objects/array
https://www.w3schools.com/js/js_arrays.asp

Reply:
i don't want people to think of sequences as arrays. there are a few overlapping scenarios between the two, but they are different in a number of important ways

Reply:
I agree with the desire to differentiate from arrays. I spent a lot of time chasing the white rabbit of sequence-backed arrays, and it was all folly. :) That said, I think you may need more words (not necessarily here) to convey the differences. You might only be able to communicate "thinking of sequences as arrays is dangerous; that way be dragons!" here, but point to a longer section of documentation titled "Sequences and Arrays" that explains these issues in more detail.

Reply:
> Specifically, i think the term index implies something that can be relied upon to index into the structure, however, this is not the case with sequences, which is why i've refered to them as positions.

Consider saying that here. "We refer to these as positions rather than indices because you cannot index into a sequence using its position like you can with an array. See _Sequences and Arrays_ for more information."